openedx/features/content_type_gating/tests/test_models.py

```diff
@@ -69,34 +66,26 @@ def test_enabled_for_enrollment(
         else:
             existing_enrollment = None
 
-        if pass_enrollment:
```

Comment:
Why did you get rid of the tests that hit the codepath that didn't pass in the enrollment?

Reply:
it appears that currently, the only case where we passed the enrollment was your code for the experiment which I am removing. Given that using the attribute without select_related might be suboptimal it seemed that just removing the unused argument would make sense (and make the interface simpler)

Reply:
Makes sense.